dl/dl-base/3.1.0/Dockerfile-py2.gpu

```diff
@@ -32,10 +36,17 @@ RUN pip --no-cache-dir install \
         tqdm \
         wheel \
         kaggle-cli \
```

Comment:
do we still need kaggle-cli?

Reply:
I think that we should remove it, is it ok for you?

Reply:
let's remove it since the official package already includes the cli